srcs/runners_manager/runner/RedisManager.py

```diff
@@ -46,6 +46,16 @@ def get_runner(self, name: str) -> Runner:
         data = json.loads(self.redis.get(name))
         return Runner.fromJson(data)
 
+    def check_runners(self, manager_name: str) -> bool:
+        """
+        Check if a runner exist or not for a specific manager_name
+        """
+        runners: Runner = self.get_all_runners()
+        for runner in runners:
+            if manager_name == runner.name:
+                return True
+        return False
+
```

Comment:
For a given manager name or for a given runner name?
```suggestion
    def check_runners(self, name: str) -> bool:
        """
        For a given runner name, check if it is managed by the service.
        """
        runners: Runner = self.get_all_runners()
        for runner in runners:
            if name == runner.name:
                return True
        return False

```

Reply:
You're right